build/ci.sh

```diff
@@ -24,15 +24,17 @@ function test {
 sudo apt-get update
 
 ## Determine version number format for CLANG/LLVM packages
+export LLVM=${LLVM:-9}
```

Comment:
Can you explain the intent here?

Reply:
`LLMV` is a required parameter for that script. I.e. it's impossible to have a script run without LLVM being assigned to some reasonable value. Although the `export` part here is an overkill. I'll rework this line, thanks for pointing out!